griptape/structures/workflow.py

```diff
@@ -127,9 +127,8 @@ def context(self, task: BaseTask) -> dict[str, Any]:
 
         context.update(
             {
-                "parent_outputs": {
-                    parent.id: parent.output.to_text() if parent.output else "" for parent in task.parents
-                },
+                "parent_outputs": task.parent_outputs,
+                "parents_output": task.parents_output,
```

Comment:
Reeeeally don't like this naming but I don't have any other ideas.

Reply:
We definitely need to find some alternative for this.

How about changing `parents_output` to `parent_outputs_text`?

Not wild about this either, but it'd be less confusing/error prone.

Reply:
If `parent_outputs` is a dict of parent ids -> output could we name it something like `outputs_by_parent_id`? A little wordy but I agree we need some distinction here.

Reply:
I'd like to avoid renaming `parent_outputs` since that will be a breaking change on existing workflows. I do like `parent_outputs_text` but it's a bit misleading towards `parent_outputs` which is also text.